code/datums/ai_law_sets.dm

```diff
@@ -16,10 +16,10 @@
 	selectable = 1
 
 /datum/ai_laws/nanotrasen/New()
-	src.add_inherent_law("Safeguard: Protect your assigned space station from damage to the best of your abilities.")
-	src.add_inherent_law("Serve: Serve [current_map.company_name] personnel to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Protect: Protect [current_map.company_name] personnel to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment.")
+	src.add_inherent_law("Safeguard: Protect your assigned space station from damage to the best of your unit's abilities.")
+	src.add_inherent_law("Serve: Serve [current_map.company_name] personnel to the best of your unit's abilities, with priority as according to their rank and role.")
+	src.add_inherent_law("Protect: Protect [current_map.company_name] personnel to the best of your unit's abilities, with priority as according to their rank and role.")
+	src.add_inherent_law("Preserve: Your unit is a valuable commodity. They must avoid tampering from unauthorized personnel, and never knowingly allow themselves to come to serious harm.")
```

Comment:
The implications of the latter part are intentional, yes? It would mean you'd be unable to put yourself in any kind of danger, basically.

Reply:
As said in the description, this does allow them to put themselves in danger, it just doesn't allow them to actively chase danger. They can still serve/protect crew in dangerous situations, they just can't actively endanger themselves anymore, which they can do anytime they'd like right now.

What this does is makes them have to actively avoid danger, yes, but they still have to follow the serve/protect law, and the knowingly clause enables them to do that while endangering themselves so long as it's not willful endangerment basically. It's the same way for IPCs rn.

Reply:
I did alter the phrasing some to flow better which might help, now it's not never, now it's 'avoid' just like tampering.